libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -136,6 +136,7 @@ class _Config:
             arbitrary_types_allowed = True
 
         include = include or []
+        include = [i for i in include if i != "configurable"]
```

Comment:
could we make it possible to generate schema that does not include "configurable" data?

Reply:
how is that useful?

Reply:
opt in to expose configuration params twice seems overkill no?

Reply:
* either we do that or that key is treated differently from other top level keys of the RunnableConfig (which is ifne)
* Any situations where a server picks up an existing configurable runnable without knowing that it's configurable and not wanting to expose configurability by default?